### ISSUE TITLE
Ambiguous NFR: Game Physics SRS - High Performance

### ISSUE THREAD

**Issue by sorainsm** [JacquesCarette/Drasil#1896]
@smiths what is a "short response time" in the context of this soft real time system? More clarification would be helpful.

**Comment by smiths** [comment]
Yes, @sorainsm, you are correct.  The NFRs for the Game Physics example were not completed.  As given, this NFR, and the others, are not adequate.  They will be improved in the future (using the ideas we will be discussing in 741).  😄 

In the future, when you create an issue, please also include a screen shot of the "offending" text.  It makes it much easier for the people responding to the issue if all of the details are already available in the issue itself.
